scheduler/config/settings.py

```diff
@@ -29,7 +29,7 @@ class Settings(BaseSettings):
     host_bytes_user: str = Field(..., env="BYTES_USERNAME")
     host_bytes_password: str = Field(..., env="BYTES_PASSWORD")
     host_octopoes: str = Field(..., env="OCTOPOES_API")
-    host_scan_profile: str = Field(..., env="SCHEDULER_RABBITMQ_DSN")
+    host_mutation: str = Field(..., env="SCHEDULER_RABBITMQ_DSN")
```

Comment:
Not necessarily for this PR, but I've been wondering why not choose the `AmqpDsn` and available URL types over `str`s? Perhaps this would be a good time to start using these stricter fields. They not only provide a better understanding of the setting, but also security and validation (to prevent unnecessary misconfigurations)

Reply:
Thanks, yeah I'll have a look into it.